notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -472,33 +472,16 @@
       },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        # pp.pprint(resp)
</pre><p>this line can be removed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/879/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
Done!